### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_Update Target pop up_AI4D: Name of the "Image" is not correct i.e. "Maininstruction".

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6666]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS; #A11yTCS; #A11y_ADS_Schema_Dacpac_Backup; #Azure Data Studio; #DesktopApp; #Win10; #AI4D; #MAS4.1.2; #MAS1.3.1;

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.9.0 (user setup)
Commit: 78a42e1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362

**Additional Details :**
Tools: AI4D.
Mas reference: MAS 4.1.2, MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Navigate till Select Source or Select Target & hit enter.
9. Provide all the details for Source & Target. In target select any database.
10. Navigate till Ok button & hit enter.
11. Navigate till Compare menu from Schema compare Menu bar & hit enter.
12. Navigate till Apply button from Schema Compare Menu bar & hit enter.
13. Update Target pop up will appear
14. Open AI4D and check the name of the image ""Main instruction".

**Actual:**
Name of the "Image" is not correct i.e. "Main instruction".

**Expected:**
Name of the "Image" should be provided as "Warning icon".

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screenreader user will receive wrong information about the image.

**MAS Reference:**
MAS 4.1.2 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

MAS1.3.1-[https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

**Attachment for Reference:**
![6666_ Accessibility A11y_AzureDataStudio(Schema,Dacpac,Backup)_Update Target pop up_AI4D](https://user-images.githubusercontent.com/24580570/62779834-e1c18c80-bad1-11e9-845f-97e78a4d6b4e.png)

**Comment by hannahqin** [comment]
Edited typo in original comment: "Waring icon" --> "Warning icon"

**Comment by Benjin** [comment]
Upstream bug.  Filed as https://github.com/microsoft/vscode/issues/79072

**Comment by Benjin** [comment]
Update: upstream even of vscode.  Filed to Election as https://github.com/electron/electron/issues/19784

**Comment by hannahqin** [comment]
Marking this for Exclusion. As stated above, this is an upstream issue with Electron and is being tracked here: https://github.com/electron/electron/issues/19784.


**Importance of Experience:** minimal
Warning popups may appear in scenarios where we want to guarantee that the user's previous action was intended. In this case, it was to ensure they want to 'Apply' changes to a Target database, which would overwrite any existing data. These warning scenarios occur infrequently.

**Quality of Experience:** not impacted
Users can receive the full context of the popup through its text contents. The context of the Warning Icon image is not necessary to understand the popup's purpose.

**AAT Conformance Doc:** https://cela-ra-aat-prod.azurewebsites.net/#/workflow/ee5c9007-47f4-4fcb-a4d7-a998a617edde?tab=default

**@fsteffi, please work with a SME to review.**

**Comment by miniak** [comment]
Unfortunately this seems to be an issue with the `TaskDialogIndirect` Windows API, that Electron uses to implement `dialog.showMessageBox()`. It cannot be fixed without significant effort of re-implementing the Electron API in a different way.

**Comment by kenvanhyning** [comment]
Closing as upstream

**Comment by Alexandria-R** [comment]
Approved as non blocking.
